src/microbe_stage/MicrobeHUD.cs

```diff
@@ -649,7 +664,9 @@ private void UpdateHoverInfo()
             microbeText.Valign = Label.VAlign.Center;
             hoveredItems.AddChild(microbeText);
 
-            microbeText.Text = "Cell of species " + entry.Species.FormattedName;
+            // Cell of species {0}
+            microbeText.Text = string.Format(CultureInfo.CurrentCulture,
+                TranslationServer.Translate("MICROBE_HUD_CELLS"), entry.Species.FormattedName);
```

Comment:
I think I'd like here the string to be more what it is about than where it is `CELL_OF_SPECIES`